src/main/java/com/google/cloud/tools/action/app/config/DefaultDeployConfiguration.java

```diff
@@ -32,16 +31,16 @@
   private final Collection<Path> deployables;
   private final String bucket;
   private final String dockerBuild;
-  private final Boolean force;
+  private final boolean force;
```

Comment:
boolean defaults to false, Boolean defaults to null, I think the behavior might be more meaningful like this.